src/migrations/20250206180000-eo-close-goals-and-objectives.js

```diff
@@ -0,0 +1,154 @@
+const { prepMigration } = require('../lib/migration');
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface) {
+    await queryInterface.sequelize.transaction(async (transaction) => {
+      const sessionSig = __filename;
+      await prepMigration(queryInterface, transaction, sessionSig);
+      return queryInterface.sequelize.query(`
+        -- This Completes any remaining equity Objectives and Closes any
+        -- remaining equity Goals
+
+        -- Create a list of EO-impacted Goals that are not yet closed
+        DROP TABLE IF EXISTS equity_goals;
+        CREATE TEMP TABLE equity_goals
+        AS
+        SELECT
+          g.id gid,
+          LEFT(name, 30) short_gname,
+          COALESCE(NULLIF(POSITION(' dei' IN LOWER(name)),0),POSITION(' equit' IN LOWER(name))) gissue_loc,
+          status g_status
+        FROM "Goals" g
+        LEFT JOIN "GoalTemplates" gt
+          ON g."goalTemplateId" = gt.id
+        WHERE (
+            LOWER(name) LIKE '% dei%'
+            OR
+            LOWER(name) LIKE '% equit%'
+            OR
+            LOWER(name) LIKE 'dei%'
+            OR
+            LOWER(name) LIKE 'equit%'
+          )
+          AND g.status != 'Closed'
+          AND g."deletedAt" IS NULL
+        ;
+
+        -- Create a list of EO-impacted Objectives that are not yet closed
+        -- and are on a non-deleted Goal. 
+        DROP TABLE IF EXISTS equity_objectives;
+        CREATE TEMP TABLE equity_objectives
+        AS
+        SELECT
+          o.id oid,
+          "goalId" ogid,
+          title full_otitle,
+          LEFT(title, 30) short_otitle,
```

Comment:
what is this for?

Reply:
It's basically leftover diagnostic fields just in case anyone wants to look at the tables and get context.